Bottom: fa1eec5fa4fe4b1b52f28c77f055b6791b5b8ab6
Top:    0449301a511d37b585d312ac34d727ba2987e824
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-08 14:08:27 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index 27e244e..2165147 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -611,6 +611,8 @@ int connect(int socket, const struct sockaddr *addr, socklen_t addrlen)
 {
 	int fd, ret;
 
+	fprintf(fout, "%d connect %d\n", (int)syscall(SYS_gettid), socket);
+	fflush(fout);
 	switch (fd_get(socket, &fd)) {
 	case fd_fork:
 		return fork_active(socket, addr, addrlen);
